.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "vets-api native setup",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/ruby-asdf:0": {
+			"version": "3.2.3"
+		},
+		"ghcr.io/itsmechlark/features/redis-server:1": {
+			"version": "7"
```

Comment:
We're currently on redis 4 and 5 iirc. We'll be updating to version 6 next week. Not sure when we'll be on 7 though.

Reply:
Cool, I changed to redis 6 which appears to work fine.